libcontainer/cgroups/fs/memory.go

```diff
@@ -170,6 +170,11 @@ func (s *MemoryGroup) GetStats(path string, stats *cgroups.Stats) error {
 		return err
 	}
 	stats.MemoryStats.SwapUsage = swapUsage
+	stats.MemoryStats.SwapOnlyUsage = cgroups.MemoryData{
+		Usage:   swapUsage.Usage - memoryUsage.Usage,
+		Limit:   swapUsage.Limit,
```

Comment:
Did I miss sth? I remember that `swapUsage.Limit` is from `memory.memsw.limit_in_bytes`? So maybe we should set `SwapOnlyUsage.Limit` with `swapUsage.Limit - memoryUsage.Limit`?

Reply:
Byt maybe we should set `SwapOnlyUsage.Limit` to 0? Because there is no swap usage limit in cgroup v1. 😢

Reply:
Theoretically, when memory usage is 0, the swap usage limit will be limit(mem+swap). But it's a really critical scene. wdyt of setting it to 0, or setting it to limit(mem+swap).

Reply:
I just revert the limit setting, pls check.